### PR TITLE
Use modal

### PR DESCRIPTION
**Pull request by IrynaKamenska**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+build/
+!**/src/main/**/build/
+!**/src/test/**/build/
+
+### VS Code ###
+.vscode/
```
Comment (ZeshanShahid79) [inline .gitignore:33]: add a new line at the end

File: frontend/src/BookOverview.tsx [hunk 2]
```diff
@@ -15,24 +17,57 @@ function BookOverview() {
...
+                    <button type={"submit"} onClick={openModal}>New Book</button>
+                </div>
+
+        }
+
     </>;
 }
 
+
 export default BookOverview;
```
Comment (ZeshanShahid79) [inline frontend/src/BookOverview.tsx:73]: same here

File: frontend/src/CreateBookModal.tsx [hunk 3]
```diff
@@ -89,6 +102,6 @@ export default function CreateBook() {
             <br/><br/>
             <button>add new book</button>
         </form>
-
-    </>
+        </Modal>
+    );
 }
```
Comment (ZeshanShahid79) [inline frontend/src/CreateBookModal.tsx:107]: same here

Comment (ZeshanShahid79) [reply frontend/src/CreateBookModal.tsx:107]: maybe also check formatting on each file(simply just do : (option+cmd+L))

**Review by ZeshanShahid79 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (IrynaKamenska) [reply .gitignore]: done

File: frontend/src/CreateBookModal.tsx (hunk 3 above)
Comment (IrynaKamenska) [reply frontend/src/CreateBookModal.tsx]: done

**Review by ZeshanShahid79 — APPROVED** [review]
great job

**Pull request merged**